### PR TITLE
Feature/registry creation form

### PR DESCRIPTION
**Pull request by jake-kent**

### CODE REVIEW

File: Maco/accounts/functions.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+from Maco.wishlists.constants import ADDRESS_TYPES  # NOQA
+
+
+def registry_creation_form_save(form):
```
Comment (gone) [inline Maco/accounts/functions.py]: Would put this on the form as save

File: Maco/accounts/views.py [hunk 2]
```diff
@@ -82,6 +84,27 @@ def get_context_data(self, **kwargs):
...
+        return self.request.user.account_type == UC.MilitaryUser
+
+
+class RegistrySetup(LoginRequiredMixin, UserPassesTestMixin, FormView):
+    template_name = "accounts/registry_creation_page.jinja"
+    success_url = reverse_lazy('profile-view')
+    form_class = RegistrySetupForm
+
+    def get_form(self, form_class=RegistrySetupForm):
+        if self.request.POST:
```
Comment (gone) [inline Maco/accounts/views.py]: Use https://ccbv.co.uk/projects/Django/2.0/django.views.generic.edit/FormView/#get_initial and https://ccbv.co.uk/projects/Django/2.0/django.views.generic.edit/FormView/#get_form_kwargs vs get form

File: Maco/wishlists/migrations/0002_auto_20160111_1108.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```
Comment (gone) [inline Maco/wishlists/migrations/0002_auto_20160111_1108.py]: Squash migrations?

Comment (jake-kent) [reply Maco/wishlists/migrations/0002_auto_20160111_1108.py]: there might be an issue with the existing oscar migrations being squashed, but have squashed and fingers crossed

**Pull request merged**